recipes/enchant/all/CMakeLists.txt

```diff
@@ -0,0 +1,123 @@
+cmake_minimum_required(VERSION 3.2)
```

Comment:
is it taken from somewhere, or written specifically for the recipe?

Reply:
I wrote this with reason https://github.com/conan-io/conan-center-index/pull/9695#discussion_r823721901